src/app/character-tile/CharacterHeaderXP.m.scss

```diff
@@ -0,0 +1,27 @@
+@import '../variables.scss';
+
+$xp-bar-height: 2px;
+
+.xp-bar {
+  position: absolute;
+  width: 100%;
+  bottom: 0;
+
+  .level-bar {
```

Comment:
Couple notes on css modules:

1. Avoid `kebab-case` in favor of `camelCase` so they look the same in JS.
2. There's no real need to nest in CSS modules, since you aren't scoping the classes to avoid conflicts. So you can flatten things out a bunch.

Reply:
ah cool! just copied the styles, I'll rename them to be camelCase, makes sense